--- http.py	2019-05-03 15:07:52.591411824 -0700
+++ http_new.py	2019-05-03 15:09:23.470304022 -0700
@@ -1897,7 +1897,7 @@ def build_http():
         http_timeout = socket.getdefaulttimeout()
     else:
         http_timeout = DEFAULT_HTTP_TIMEOUT_SEC
-    http = httplib2.Http(timeout=http_timeout)
+    http = set_user_agent(httplib2.Http(timeout=http_timeout), '-kfpipeline-')
     # 308's are used by several Google APIs (Drive, YouTube)
     # for Resumable Uploads rather than Permanent Redirects.
     # This asks httplib2 to exclude 308s from the status codes
